tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -555,7 +556,9 @@ class DxcIncludeHandlerForInjectedSources : public IDxcIncludeHandler {
     _COM_Outptr_result_maybenull_ IDxcBlob **ppIncludeSource
   ) override {
     try {
-      *ppIncludeSource = includeFiles.at(std::wstring(pFilename));
+      std::wstring FilenameStr(pFilename);
+      std::replace(FilenameStr.begin(), FilenameStr.end(), '/', '\\');
```

Comment:
As long as we are normalizing file paths, why not pick / as the path separator? That works on both windows and linux.

Reply:
That's a good suggestion and one that I considered. Mostly I was nervous about a change that might jeopardize file matching as we've seen a few failures in that area recently. I'd need to look more into where those paths originate.